aptos-move/framework/aptos-framework/sources/randomness.move

```diff
@@ -195,12 +195,20 @@ module aptos_framework::randomness {
 
         let sample = r1 % range;
         let i = 0;
-        while (i < 256) {
+        while ({
+            spec {
+                invariant sample >= 0 && sample < max_excl - min_incl;
+            };
```

Comment:
i assume these are extra specs amd do not contribute to the proof of reconfiguration?

Reply:
They are used to prove properties inside randomness.move